apps/demo.namerank.io/app/page.tsx

```diff
@@ -0,0 +1,27 @@
+import { Suspense } from "react";
+
+import { Form } from "./form";
+import Results from "./results";
+import { Skeleton } from "./skeleton";
+
+interface HomeProps {
+  searchParams?: Promise<{
+    name?: string;
```

Comment:
rename to label

Reply:
Well the value being input by the user is a name, and then we should internally be transforming that into the `labelForAnalysis` as you're doing.